#MultiArrayLayout  layout
float64[]         rupee_trace
float64[]         rupee_trace_normalized
